GreysonElkins/exquisite-corpse
# Feature/cc welcome page basics

#### What's this PR do?
* Renders the Welcome Page with instructions, history, current stories, and published story components
* styles the Welcome page with basic styling 
* Tests that the correct things are rendered on the page 
* tests that when buttons are clicked, the necessary behavior occurs

#### Where should the reviewer start?
WelcomePage.js

#### How should this be manually tested?
 run `npm test` in the testing suite.
and 
Open up browser to see the changes on the main home page

#### Any background context you want to provide?
There is mock data currently for the prompts that will be replaced by api data

#### What are the relevant tickets?
closes #12 

#### Screenshots (if appropriate)

<img width="726" alt="Screen Shot 2020-09-12 at 5 54 17 PM" src="https://user-images.githubusercontent.com/61163743/93007038-0c75e400-f521-11ea-88b7-1e6eda1a13e6.png">
<img width="720" alt="Screen Shot 2020-09-12 at 5 54 24 PM" src="https://user-images.githubusercontent.com/61163743/93007039-0f70d480-f521-11ea-938e-709d499d26cd.png">
<img width="1197" alt="Screen Shot 2020-09-12 at 5 54 31 PM" src="https://user-images.githubusercontent.com/61163743/93007041-11d32e80-f521-11ea-8eff-01562fc8f38a.png">
